lib/ddtrace/contrib/sinatra/tracer.rb

```diff
@@ -42,6 +42,9 @@ def render(engine, data, *)
                   # want to record it.
                   span.set_tag(Ext::TAG_TEMPLATE_NAME, data) if data.is_a? Symbol
 
+                  # Measure service stats
+                  Contrib::Analytics.set_measured(span)
```

Comment:
same here for action view, do we want to add metrics for template rendering?

Reply:
or well, the question is partially "do we do it today?" + "does it make sense to add it going forward?"